packages/scaffolding-cli/templates/src/ssr/components/LocaleSwitcher/index.tsx

```diff
@@ -16,13 +16,19 @@ export const LocaleSwitcher: React.FC = () => {
 
     const [lang, setLang] = React.useState("en-GB")
```

Comment:
Is this always going to be hardcoded? Should we have this as a config option in a file with a default? just because Contentful is default en-US and might trip up some people in knowing how to ensure that the locals are in sync.

Reply:
Yes. You are right. I was going to add it to the config or even for future add it to advance-cli options. Good shout.